.travis.yml

```diff
@@ -21,7 +21,6 @@ jdk:
   - oraclejdk9
   - oraclejdk8
   - openjdk7
-
```

Comment:
Do not strip trailing spaces or remove empty lines on the existing code as part of your PR.